src/component/tooltip/TooltipContent.js

```diff
@@ -250,11 +253,23 @@ TooltipContent.prototype = {
         if (domStyle.position !== 'absolute' && stl.position !== 'absolute') {
             domStyle.position = 'relative';
         }
+        this._moveTooltip()
```

Comment:
Maybe should add a judgement about whether `alwaysShowContent` is `true`
`TooltipView.js`([Line 112](https://github.com/apache/incubator-echarts/blob/master/src/component/tooltip/TooltipView.js#L112))
```js
tooltipContent.update(tooltipModel);
```
Pass the parameter `tooltipModel` to `update` function.
```js
var alwaysShowContent = tooltipModel.get('alwaysShowContent');
alwaysShowContent && this._moveTooltip();
```

Reply:
> Maybe should add a judgement about whether `alwaysShowContent` is `true`
> `TooltipView.js`([Line 112](https://github.com/apache/incubator-echarts/blob/master/src/component/tooltip/TooltipView.js#L112))
> 
> ```js
> tooltipContent.update(tooltipModel);
> ```
> 
> Pass the parameter `tooltipModel` to `update` function.
> 
> ```js
> var alwaysShowContent = tooltipModel.get('alwaysShowContent');
> alwaysShowContent && this._moveTooltip();
> ```

ok i see. thank you!